### PR TITLE
ItemizedLayer: Added a check that prevents Gestures from being processed if the Layer is not enabled

### PR DESCRIPTION
**Pull request by carlos-mg89**

### CODE REVIEW

File: vtm/src/org/oscim/layers/marker/ItemizedLayer.java [hunk 1]
```diff
@@ -264,11 +264,13 @@ protected boolean activateSelectedItems(MotionEvent event, ActiveItem task) {
 
     @Override
     public boolean onGesture(Gesture g, MotionEvent e) {
-        if (g instanceof Gesture.Tap)
-            return activateSelectedItems(e, mActiveItemSingleTap);
+        if (isEnabled()) {
```
Comment (devemux86) [inline vtm/src/org/oscim/layers/marker/ItemizedLayer.java]: For simplicity and better code reading, can do the check at method's start:
```java
if (!isEnabled())
    return;
```

**Comment by carlos-mg89** [comment]
Done @devemux86 !

File: vtm/src/org/oscim/layers/marker/ItemizedLayer.java [hunk 2]
```diff
@@ -264,6 +264,9 @@ protected boolean activateSelectedItems(MotionEvent event, ActiveItem task) {
 
     @Override
     public boolean onGesture(Gesture g, MotionEvent e) {
+        if (!isEnabled())
+            return;
```
Comment (devemux86) [inline vtm/src/org/oscim/layers/marker/ItemizedLayer.java]: Should be:
```java
return false;
```

**Comment by devemux86** [comment]
Thanks!

Squashed and merged via 49d1e7bafc8e62d66b03af0d9022fba77ea99829.

**Pull request closed without merging**